## Reposition Request Service

# Request fields
geometry_msgs/Pose pose
---

# Response fields
bool success  # true if the reposition happens without exception, false otw